packages/core/src/components/Tooltip.svelte

```diff
@@ -57,6 +58,21 @@
 
   const tooltipId = uuidv4();
```

Comment:
We could change the tooltip id to be like marker / message id: `visahoi-tooltip-contextkey-uuid`